content/docs/reference/config/authorization/constraints-and-properties/index.md

```diff
@@ -4,35 +4,46 @@ description: Describes the supported constraints and properties.
 weight: 10
 ---
 
-This page lists the supported keys that could be used in `Constraints` and `Properties`.
```

Comment:
I more than welcome your update of this page. I will leave comments on the things that also need to change in the text.